src/zh/zazhimi/src/eu/kanade/tachiyomi/extension/zh/zazhimi/Zazhimi.kt

```diff
@@ -0,0 +1,102 @@
+package eu.kanade.tachiyomi.extension.zh.zazhimi
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import keiyoushi.utils.parseAs
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+
+class Zazhimi : HttpSource() {
+
+    override val baseUrl = "https://android2024.zazhimi.net/api"
+    override val lang = "zh"
+    override val name = "杂志迷"
+    override val supportsLatest = false
+
+    override fun headersBuilder() = super.headersBuilder()
+        .set("User-Agent", "ZaZhiMi_5.3.0")
+
+    // Popular
+
+    override fun popularMangaRequest(page: Int) = GET("$baseUrl/index.php?p=$page&s=20", headers)
+
+    override fun popularMangaParse(response: Response): MangasPage {
+        val result = response.parseAs<IndexResponse>()
+        return MangasPage(result.new.map(NewItem::toSManga), true)
```

Comment:
Since the site does not indicate if there are any more results you should at least set `hasNextPage` to check if you get less than the requested entries or not.